src/tests/integration/server.itest.js

```diff
@@ -513,6 +513,45 @@ describe('/api/monsters', () => {
       expect(res.body.results[0].name).toEqual(name);
     });
   });
+  describe('with challenge rating query', () => {
+    it('returns objects with only one provided challenge rating', async () => {
+      const res = await request(app).get('/api/monsters?challenge_rating=0.25');
+      expect(res.statusCode).toEqual(200);
+
+      const indexRes = await request(app).get(`/api/monsters/${res.body.results[0].index}`);
```

Comment:
**[Nitpick]**: For this and the one in the test below, I'd pick a random index from the count of results. This could make the test flaky but it being flaky is a good sign that something has gone terribly wrong.

Reply:
👍

Reply:
Fixed